### PR TITLE
OSDOCS-8487 update installer RN tech preview table

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-8487

Link to docs preview:
https://71794--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-technology-preview-tables

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by bscott-rh** [comment]
@MayXuQQ PTAL at this update to the 4.15 release notes tech preview table. TY

**Review by barbacbd — COMMENTED** [review]
/lgtm

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -1477,6 +1477,16 @@ In the following tables, features are marked with the following statuses:
 |====
 |Feature |4.13 |4.14 |4.15
 
+|Installing {product-title} on Oracle Cloud Infrastructure with VMs
```
Comment (skopacz1) [inline release_notes/ocp-4-15-release-notes.adoc:1480]:
```suggestion
|Installing {product-title} on {oci-first} with VMs
```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -1477,6 +1477,16 @@ In the following tables, features are marked with the following statuses:
...
+|N/A
+|Developer Preview
+|Technology Preview
+
+|Installing {product-title} on Oracle Cloud Infrastructure on baremetal
```
Comment (skopacz1) [inline release_notes/ocp-4-15-release-notes.adoc:1485]:
```suggestion
|Installing {product-title} on {oci} on baremetal
```
Or if you'd prefer to have the Oracle name fully written out in this case for better clarity, you could probably use `{oci-first}` again.

Comment (skopacz1) [inline release_notes/ocp-4-15-release-notes.adoc:1485]:
```suggestion
|Installing {product-title} on Oracle Cloud Infrastructure on bare metal
```

**Review by skopacz1 — COMMENTED** [review]
Left a few nits but otherwise LGTM!

**Pull request merged**